### DISCUSSION TITLE
Can we update the emmc boot0/boot1 partition using rauc?

### DISCUSSION THREAD

**Discussion by pranav580** [rauc/rauc#1476] [Q&A] [answered]
i want to update the eMMC mmcblk0boot0 and mmcblk0boot1 partition as an A/B scheme,
currently i am using this system.conf
```
[system]
compatible=demo-x86
bootloader=uboot
mountprefix=/mnt/rauc
bundle-formats=-plain

[keyring]
path=rauc.cert.pem

[slot.bootloader.0]
device=/dev/mmcblk0boot0
type=boot-emmc
bootname=system0

[slot.bootloader.1]
device=/dev/mmcblk0boot1
type=boot-emmc
bootname=system1

[slot.rootfs.0]
device=/dev/mmcblk0p1
type=ext4

[slot.rootfs.1]
device=/dev/mmcblk0p2
type=ext4
```

How should i build the bundle

system configurations: TI am6411B
eMMc size: 16GB
partitions: p1-1GB, p2-1GB

**Comment by jluebbe** [comment, marked as answer]
You shouldn't have two slots for the bootloader, as there is no A/B fallback for the bootloader. There is an example in the documentation: https://rauc.readthedocs.io/en/latest/advanced.html#update-bootloader-in-emmc-boot-partitions

Also, the bootloader slot shouldn't have a bootname (that belongs on the rootfs), as the A/B decision concerns the rootfs.

The bundle just needs an image for the bootloader:
```
[image.bootloader]
filename=my-bootloader.img
```

If you're using Yocto, perhaps take a look at https://github.com/linux-automation/meta-lxatac/blob/scarthgap/meta-lxatac-software/recipes-core/bundles/lxatac-core-bundle-base.bb#L24 and https://github.com/linux-automation/meta-lxatac/blob/scarthgap/meta-lxatac-bsp/recipes-core/rauc/files/lxatac/system.conf
